sigridci/sigridci.py

```diff
@@ -52,7 +52,7 @@ def callSigridAPI(self, api, path):
         request = urllib.request.Request(url, None)
         request.add_header("Accept", "application/json")
         request.add_header("Authorization", \
-            b"Basic " + base64.urlsafe_b64encode(f"{self.account}:{self.token}".encode("utf8")))
+            b"Basic " + base64.standard_b64encode(f"{self.account}:{self.token}".encode("utf8")))
```

Comment:
Looks good to me!

Reply:
@bruno-oliveira then feel free to merge it :)